modules/publisher/views.py

```diff
@@ -149,20 +139,10 @@ def post_account_name():
         errors = []
         try:
             api.post_username(flask.session, username)
-        except ApiTimeoutError as api_timeout_error:
-            flask.abort(504, str(api_timeout_error))
-        except ApiConnectionError as api_connection_error:
-            flask.abort(502, str(api_connection_error))
-        except ApiResponseDecodeError as api_response_decode_error:
-            flask.abort(502, str(api_response_decode_error))
         except ApiResponseErrorList as api_response_error_list:
             errors = errors + api_response_error_list.errors
```

Comment:
Why is not `_handle_errors_list` used here as well?

Reply:
same answer